suite-native/navigation/src/types.tsx

```diff
@@ -64,3 +68,7 @@ export type TabsOptions = {
         params?: Record<string, unknown>;
     };
 };
+
+export type NavigateParameters<TParamList extends ParamListBase = ParamListBase> = Parameters<
+    NavigationHelpers<TParamList>['navigate']
+>;
```

Comment:
It's probably not the most elegant way, I don't want to spend too much time on it but if you use the type of first item in the array, you don't have to wrap it in an array and then destructuring (...onCancelNavigationTarget)...
```suggestion
export type NavigateParameters<TParamList extends ParamListBase = ParamListBase> = Parameters<
    NavigationHelpers<TParamList>['navigate']
>[0];
```

Reply:
https://github.com/trezor/trezor-suite/commit/faf9766feb14f843bc6760e496a14cedb8586b25